### PR TITLE
[WIP] Fix Tree component and add storybook

### PR DESCRIPTION
**Pull request by nchevobbe**

### CODE REVIEW

File: packages/devtools-components/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+      webpackConfig.resolve.alias.react = "react/lib/ReactUMDEntry";
+    }
+  }
+
+  webpackConfig.node = { fs: "empty" };
+
+  webpackConfig.plugins = webpackConfig.plugins || [];
+  webpackConfig.plugins.push(
+    new webpack.DefinePlugin({
+      "process.env": {
```
Comment (jasonLaster) [inline packages/devtools-components/webpack.config.js]: do we need the define plugin?

File: packages/devtools-components/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    })
+  );
+
+  if (isDevelopment()) {
+    webpackConfig.module.rules.push({
+      test: /\.css$/,
+      use: [
+        { loader: "style-loader" },
+        { loader: "css-loader", options: { importLoaders: 1 } },
+        // {
```
Comment (jasonLaster) [inline packages/devtools-components/webpack.config.js]: what's going on here?

File: packages/devtools-components/stories/tree.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        focused ? " (focused)" : null
+      ),
+      getRoots: () => ["A"],
+      getKey: x => "key-" + x,
+      itemHeight: 1,
+      onExpand: x => TEST_TREE.expanded.add(x),
+      onCollapse: x => TEST_TREE.expanded.delete(x),
+      isExpanded: x => TEST_TREE.expanded.has(x),
+    });
+  });
```
Comment (jasonLaster) [inline packages/devtools-components/stories/tree.js]: these tests are not run on CI,

but i'd love to get them running on CI so we can make sure they don't regress: https://github.com/mthuret/storybook-addon-specifications

File: packages/devtools-components/src/tree.js [hunk 4]
```diff
@@ -94,22 +131,6 @@ const TreeNode = createFactory(createClass({
       // unless there is an input/button child.
       dom.button(this._buttonAttrs)
```
Comment (jasonLaster) [inline packages/devtools-components/src/tree.js]: does this still exist?

File: packages/devtools-components/webpack.config.js (hunk 2 above)
Comment (nchevobbe) [inline packages/devtools-components/webpack.config.js]: I copied the webpack config from the launchpad, which is not the right thing to do :)

File: packages/devtools-components/stories/tree.js (hunk 3 above)
Comment (nchevobbe) [inline packages/devtools-components/stories/tree.js]: Yeah, I wanted to have story book so we can have at least a way to see how it looks.
This component should be properly tested as well (render, keyboard navigation, …)

File: packages/devtools-components/src/tree.js (hunk 4 above)
Comment (nchevobbe) [inline packages/devtools-components/src/tree.js]: Didn't check, I will !

**Pull request closed without merging**